inst/scripts/importPDB.R

```diff
@@ -0,0 +1,76 @@
+library("bio3d")
+library("Biostrings")
+install.packages("seqnir")
+
+readRDS(MT.RDS)
+
+
+for (genes in gene_list){
+    results <- list()
+    # Translate the gene to its aa sequence
+    targetAA <- AAString(paste(a.a. sequence of the gene, collapse="")
+
+    for (PDB-ID/chains matched to gene){
+        # vectors for the data.frame that will be created at the end of this for loop.
+        rowNames <- c()
+        resIDs <- c()
+        aaType <- c()
+        sidechainXYZ <- c()
+        calphaXYZ <- c()
+
+        # load the PDB file.
+        subject <- read.pdb(PDB-ID)
+        # get the aa sequence of the subject
+        subjectAA <- AAString(paste(PDB$seqres, collapse=""))
+        #Perform pairwisealignment..
+        ali <- pairwiseAlignment(subjectAA, targetAA)
+        # ^^^ Q: what should the gapExtension and gapOpening penalties be?
+        for (i in 1:nmatch(ali)){   #For all matched residues:
+            i <- index of residue in targetAA
+            j <- index of residue in subjectAA
+            #^^^ Q: How do I get the indices of residues aligned in target and subject?
+            rowNames <- append(rowNames, i)     # rowname will be the index of the residue in target gene
+            resIDs <- append(unique(subject$atom[subject$atom$resno==j,]$resid)) #resids are the 3 letter code of the aa
```

Comment:
Well the description of GeneData$3D on the task page says "Columns are the the **structure residue ID**, **amino acid type**, and the x, y, and z coordinates of the Cα atom and of the side-chain centroid.